lib/registries/addon/components/registration-form-view/x-file/template.hbs

```diff
@@ -1,9 +1,11 @@
-<div local-class='Files'>
```

Comment:
Not sure what the changes here were for? `.Files` flex class renders the files list in a column just as unordered list `ul`. If using `ul` is less confusing, we should fix that class and add `list-style: none;` for consistency.

Reply:
I changed the "choose" questions to bulleted lists to make it clear they're not free-text responses, then changed the files list to match. So I guess there are two things:
- `<ul>` is semantically correct, so it's better for screenreaders/a11y
- a visual distinction between free-text responses and constrained lists is helpful for eyereaders

Reply:
but I can run that by product, since it's a user-visible change

Reply:
oh! I missed that the `.Files` class might be unnecessary (or at least doesn't have to be flex). will look into that